src/System.Net.Http/src/System/Net/Http/Managed/AutoRedirectHandler.cs

```diff
@@ -82,7 +82,7 @@ public AutoRedirectHandler(int maxAutomaticRedirections, HttpMessageHandler inne
                 redirectCount++;
                 if (redirectCount > _maxAutomaticRedirections)
                 {
-                    throw new HttpRequestException("max redirects exceeded");
+                    throw new HttpRequestException(SR.net_http_max_redirects);
```

Comment:
Desktop returns 3xx.  And also UWP returns 3xx now.  I was implying that .NET Core should do the same and managedhandler should follow Desktop.